py3status/modules/process_status.py

```diff
@@ -21,14 +27,14 @@
 import os
 import subprocess
 
-ERR_NO_PROCESS = 'no process name given'
-
 
 class Py3status:
     """
     """
     # available configuration parameters
     cache_timeout = 10
+    format = '{state}'
+    format_error = 'no process name given'
```

Comment:
Adding `format_error` prevent hardcoded message. (Error or not). That was the goal.

Option 1...
Rip `format_error` out completely? If no process name given, display nothing. 

Option 2...
Rip `format_error` out completely? If no process name given, display `format_not_running`

Option 2...
Replace `format_error` with `error? (or your choice of word).